extensions/amp-pan-zoom/0.1/amp-pan-zoom.css

```diff
@@ -22,4 +22,25 @@
 
 .i-amphtml-pan-zoom-child {
   position: absolute;
+}
+
+.i-amphtml-pan-zoom-button {
+  position: absolute;
+  right: 12px;
+  width: 36px;
+  height: 36px;
+  bottom: 12px;
+  background-repeat: no-repeat;
+  background-position: center center;
+  box-shadow: 1px 1px 2px;
+  background-color: white;
+  border-radius: 3px;
+}
+
+.i-amphtml-pan-zoom-in-button {
```

Comment:
I was gonna suggest making these public but I guess if they want to customize they can build their own with the actions.

Reply:
Done and documented.